docs/tutorials/docker-compose.yml

```diff
@@ -53,7 +53,7 @@ services:
 
   # Runs the Kafka KSQL data generator for topic called "pageviews"
   ksql-datagen-pageviews:
-    image: "confluentinc/ksql-examples:0.5"
+    image: "confluentinc/ksql-examples:4.1"
```

Comment:
Regarding the question I raised in the summary; is updating these to 4.1 the right thing to do? Or should these be 4.1.0-SNAPSHOT for now to support testing?

Reply:
Feels like, from your above comment, that these should also be left at 0.5?

Reply:
Yes.